src/nupic/proto/TemporalMemoryProto.capnp

```diff
@@ -28,4 +28,5 @@ struct TemporalMemoryProto {
   winnerCells @15 :List(UInt32);
   matchingSegments @16 :List(UInt32);
   matchingCells @17 :List(UInt32);
+  permanenceOrphanDecrement @18 :Float32;
```

Comment:
You can move this somewhere else if it is better for organization, just keep it as `@18`. For backwards compatibility you can't renumber the existing ones but they are intended to be out of order in the struct.